app/views/shared/_topology_header_toolbar.html.haml

```diff
@@ -4,14 +4,15 @@
     = _("Display Names")
 
 .form-group
-  %button.btn.btn-default{'data-function'      => 'miqCallAngular',
-                          'data-function-data' => {:name => 'refresh',
+  %button.btn.btn-default{'data-function'      => 'sendDataWithRx',
+                          'data-function-data' => {:controller => 'cloudTopologyController',
+                                                   :name => 'refresh',
                                                    :args => []}.to_json}
```

Comment:
you can drop that `:args => []` from everywhere, no longer needed with `sendDataWithRx`